client/src/cmdhfvas.c

```diff
@@ -199,6 +198,7 @@ static int LoadReaderPrivateKey(uint8_t *buf, size_t bufLen, mbedtls_ecp_keypair
         return PM3_EINVARG;
```

Comment:
You’re probably missing a `tlvdb_free(derRoot);` here

Reply:
Made a change to fix that